src/archivematicaCommon/lib/elasticSearchFunctions.py

```diff
@@ -610,6 +610,34 @@ def index_mets_file_metadata(client, uuid, metsFilePath, index, type_, sipName,
             xml = ElementTree.tostring(amdSecInfo)
             indexData['METS']['amdSec'] = rename_dict_keys_with_child_dicts(normalize_dict_values(xmltodict.parse(xml)))
 
+        # Get the parent division for the file pointer
+        # by searching the physical structural map section (structMap)
+        file_id = file_.attrib.get('ID', None)
+        file_pointer = root.find(
+            "mets:structMap[@TYPE='physical']//mets:fptr[@FILEID='{}']".format(
+                file_id
+            ),
+            namespaces=ns.NSMAP
+        )
```

Comment:
If the `..` syntax works, we could try to get the parent element in here ...

```
file_pointer_division = root.find(
    "mets:structMap[@TYPE='physical']//mets:fptr[@FILEID='{}']/..".format(
        file_id
    ),
    namespaces=ns.NSMAP
)
```

Otherwise, with the `FILEID` to `DMDID` mapping, we could avoid this query and pass directly to the one bellow to get the related `mets:dmdSec[@ID='{}']/mets:mdWrap/mets:xmlData` element.

Reply:
Thanks! It worked like a charm.